src/process_wrap.cc

```diff
@@ -93,39 +93,46 @@ class ProcessWrap : public HandleWrap {
   static void ParseStdioOptions(Environment* env,
                                 Local<Object> js_options,
                                 uv_process_options_t* options) {
+    Local<Context> context = env->isolate()->GetCurrentContext();
```

Comment:
Does `env->context()` work? Ditto below.

Reply:
Yes it does. Updated.